database-mysql/index.js

```diff
@@ -1 +1,33 @@
 // this is where the database connection will happen 
+const Sequelize = require('sequelize');
+
+const host = 'localhost';
+const user = 'root';
+const port = 3306;
+const password = '';
+const database = 'joke-me-off';
+
+const connection = new Sequelize(database, user, password, {
+  dialect: 'mysql',
+  host,
+  port,
+});
+
+const Users = connection.define('users', {
+    name: Sequelize.STRING,
+    user_name: Sequelize.STRING,
+    profile_pic: Sequelize.STRING.BINARY
```

Comment:
profile pic will just be a STRING not .BINARY since we're only storing image URLs not the image itself